x-pack/legacy/plugins/cross_cluster_replication/server/np_ready/routes/api/follower_index.ts

```diff
@@ -164,6 +164,18 @@ export const registerFollowerIndexRoutes = ({ router, __LEGACY }: RouteDependenc
       path: `${API_BASE_PATH}/follower_indices/{id}`,
       validate: {
         params: schema.object({ id: schema.string() }),
+        body: schema.object({
```

Comment:
It would be better to declare the schema outside the route handler so we can re-use and compose it for the `create` route.

Reply:
I've done this in https://github.com/elastic/kibana/pull/62890, so I'm going to leave it out of this PR.